htdocs/main.inc.php

```diff
@@ -48,6 +48,18 @@
 	}
 }
 
+if (!function_exists('is_countable')) {
```

Comment:
main is loaded only in web context when memory.lib can be used in cli context too.
The is_countable function should be moved into master, shouldn't it ?

Reply:
I moved function in master.inc.php